app/routers/friendview.py

```diff
@@ -0,0 +1,26 @@
+from fastapi import APIRouter, Depends, Request
+from sqlalchemy.orm import Session
+from starlette.templating import _TemplateResponse
+
+from app.dependencies import get_db, templates
+from app.internal import friend_view
+
+
+router = APIRouter(tags=["friendview"])
+
+
+@router.get("/friendview")
+def friendview(
+        request: Request,
+        db: Session = Depends(get_db),
+        my_friend: str = None,
+) -> _TemplateResponse:
+
+    user_id = 1
```

Comment:
this should go for the tests, and not be on the main app.

Reply:
I have added a TODO: Waiting for user registration comment